optimum/exporters/tflite/convert.py

```diff
@@ -173,6 +244,118 @@ def export(
     with TemporaryDirectory() as tmp_dir_name:
         model.save(tmp_dir_name, signatures=signatures)
         converter = tf.lite.TFLiteConverter.from_saved_model(tmp_dir_name)
+
+        if quantization is not None:
+            if isinstance(quantization, str) and not isinstance(quantization, QuantizationApproach):
+                quantization = QuantizationApproach(quantization)
+
+        if quantization in [QuantizationApproach.INT8, QuantizationApproach.INT8x16]:
+            converter.optimizations = [tf.lite.Optimize.DEFAULT]
+
+            # Handling the calibration dataset:
+            # - Either loading the default dataset if no calibration dataset was provided or the required dataset,
+            # - Splitting the dataset with the provided a dataset split or with the first split if none is provided.
+            # - Shuffling the split.
+            # - Selecting num_calibration_samples in the dataset split.
+            # - Batching the dataset.
+            # - Converting it to the TensorFlow format.
+            from ...exporters import TasksManager
+
+            task = TasksManager.get_task_from_model(model)
```

Comment:
Is this really needed? Couldn't we rather pass the task as an argument, and avoid adding `get_task_from_model` in the TasksManager?

Reply:
We could, but I think this is better (it is normalized, the user does not need to know if he should use `text-classification` or `sequence-classification` for instance). It is also used in ORT now since we had a similar need there.